### PR TITLE
background color documentation

### PR DESCRIPTION
**Pull request by grandleemon**

### CODE REVIEW

File: docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+{
+  id: "backgroundColor",
+  type: "backgroundColor", 
+  states: ["normal", "hover", "active"]
+}
+```
+
+#### Config opacity example
+
+Controls whether the opacity of the border can be changed.
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md:184]: if opacity slider also does not render then need to describe it here

File: docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+{
+  active: "start" | "end";
+  degree: number;
+  end: number;
+  gradientType: "linear" | "radial";
+  hex: string;
+  opacity: 1;
+  palette: string;
+  start: number;
+  type: "solid" | "gradient";
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md:76]: `gradientHex ` is missing 

to check all keys that return look at `toElementModel` here: `editor/visual/editor/js/utils/options/BackgroundColor/converters.ts`

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image

File: docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md (hunk 2 above)
Comment (grandleemon) [reply docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md:69]: added `gradientHex`, `gradientPalette, `gradientOpacity`

**Review by victor-pg — APPROVED** [review]

**Pull request merged**